pubtools/_quay/push_docker.py

```diff
@@ -439,10 +422,7 @@ def run(self):
         # Return repos for UD cache flush
```

Comment:
I don't quite understand this: I checked where we use the UD cache flush API in Pub and it's the same as we do when flushing rhsm-pulp repos.

So, given that API expects rhsm-pulp repo IDs, why would we ask it to do cache flushes for either of:

- docker-pulp repo IDs (note: this case will be removed after this refactor and associated pub change)
- external repo IDs

I suppose it's possible UD was updated to accept those types of repo IDs and I don't know about it.  On the other hand it's possible this code doesn't make sense.  Could anyone explain it?

Reply:
Note, I have https://code.engineering.redhat.com/gerrit/c/rcm-pub/+/248049 also (but still waiting for CI)